app/backend/approaches/chatreadretrieveread.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, search_client: SearchClient, chatgpt_deployment: str, gpt_dep
         self.sourcepage_field = sourcepage_field
         self.content_field = content_field
 
-    def run(self, history: list[dict], overrides: dict) -> any:
+    def run(self, history: 'list[dict]', overrides: dict) -> any:
```

Comment:
Why would this be required? If this is to fix older Python versions, I would go for the following fix:

```python3
from __future__ import annotations
```

Reply:
Sure, that is a better resolution. This was aimed to resolve #69, indeed related to older Python versions (<=3.9). Added another commit.